dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1702,18 +1895,18 @@ std::tuple<winrt::TeachingTipPlacementMode, bool> TeachingTip::DetermineEffectiv
 std::tuple<winrt::TeachingTipPlacementMode, bool> TeachingTip::DetermineEffectivePlacementUntargeted(double contentHeight, double contentWidth)
 {
     auto const windowBounds = GetWindowBounds();
-    if (ShouldConstrainToRootBounds())
+    if (!ShouldConstrainToRootBounds())
     {
         auto const screenBoundsInCoreWindowSpace = GetEffectiveScreenBoundsInCoreWindowSpace(windowBounds);
-        if (screenBoundsInCoreWindowSpace.Height < contentHeight && screenBoundsInCoreWindowSpace.Width < contentWidth)
+        if (screenBoundsInCoreWindowSpace.Height > contentHeight && screenBoundsInCoreWindowSpace.Width > contentWidth)
         {
```

Comment:
what are these changes?

Reply:
Another bug, this one with Untargetted auto placed tips, it was revealed while debugging the IsOpen set from markup case.

---
In reply to: [279094635](https://github.com/Microsoft/microsoft-ui-xaml/pull/619#discussion_r279094635) [](ancestors = 279094635)